test-framework/src/main/java/org/keycloak/test/framework/server/DistributionKeycloakTestServer.java

```diff
@@ -16,8 +20,13 @@ public class DistributionKeycloakTestServer implements KeycloakTestServer {
     private RawKeycloakDistribution keycloak;
 
     @Override
-    public void start(List<String> rawOptions) {
+    public void start(List<String> rawOptions, Set<Class<? extends TestProvider>> testProviders) {
```

Comment:
Would be better to pass in List<TestProvider> instead, and not make the server implentations have to initialise the classes.